app/components/Header.tsx

```diff
@@ -132,6 +133,15 @@ export function Header() {
                       >
                         Peer Endorsements
                       </Link>,
+                      idp === 'COGNITO' && (
```

Comment:
```suggestion
                      idp || (
```

Reply:
We're taking advantage of short circuit evaluation here.

Reply:
I tried it and that does not work. It fails to show in the upper right user nav. changing it to && does work.

Reply:
Are you sure? I don't see how that can be right. In pseudocode, this should be:

```
if idp is null:
    display reset password
else:
    display nothing
```

If written as `idp && <Component/>`, then if idp is null (which is falsey), then due to short circuit evaluation the right hand side operand (`<Component/>`) will not be evaluated (which is the wrong behavior).

Reply:
<img width="1061" alt="or_isnt_working" src="https://github.com/nasa-gcn/gcn.nasa.gov/assets/6545874/adbb762d-5c71-4ffa-8202-4a11f2162180">

Reply:
<img width="1119" alt="double_and_works" src="https://github.com/nasa-gcn/gcn.nasa.gov/assets/6545874/a1b9d823-be7c-409a-a541-aa855f8a67d3">

Reply:
Ah! some miscommunication here: I had suggested

```jsx
idp || <Component/>
```

but you went with

```jsx
!idp && <Component/>
```

which has the same effect.